src/Resources/config/event_listener.php

```diff
@@ -11,14 +11,25 @@
  * file that was distributed with this source code.
  */
 
+use Sonata\AdminBundle\EventListener\AdminEventListener;
 use Sonata\AdminBundle\EventListener\ConfigureCRUDControllerListener;
+use Sonata\AdminBundle\Request\AdminFetcherInterface;
 use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;
+use Symfony\Component\DependencyInjection\Loader\Configurator\ReferenceConfigurator;
 
 return static function (ContainerConfigurator $containerConfigurator): void {
     // Use "service" function for creating references to services when dropping support for Symfony 4.4
     // Use "param" function for creating references to parameters when dropping support for Symfony 5.1
     $containerConfigurator->services()
 
+        ->set(AdminEventListener::class)
+            ->tag('kernel.event_subscriber', [])
```

Comment:
Is there any reason to add a second parameter here?

```suggestion
            ->tag('kernel.event_subscriber')
```